mediafile.py

```diff
@@ -2101,12 +2101,68 @@ def as_dict(self):
         StorageStyle('ARTIST_CREDIT'),
         ASFStorageStyle('beets/Artist Credit'),
     )
+    artists_credits = ListMediaField(
+        MP3ListDescStorageStyle(desc=u'ARTISTSCREDITS'),
+        MP3ListDescStorageStyle(desc=u'ARTISTS_CREDITS'),
+        MP3ListDescStorageStyle(desc=u'ARTISTS CREDITS', read_only=True),
```

Comment:
It looks like you're using on-disk tag names that are similar to all the formats we use for the `artist_credit` field. The reason we adopted all of these is because of other software, i.e., we found that other tools were expecting three different formats. I suspect that this is _not_ true for the "plural" form of the field… it would be great to do a little research into what other software is using for these fields to make sure we match!

Reply:
Most software doesn't support multi-tags in general. The few that do typically only support `artists` and `albumartists` from my research. I think we're at a crossroads where beets can set the standard for expanded multi-tag support that future software adopts.

As for supporting multiple variations like this, I think it's safer for the long-run but would be okay slimming down to just one tag name.

Reply:
OK, thanks! With this in mind, I think it would be a really good idea to support only a single variant. If MediaFile really is the first project in the world to support "multi" variants of these tags, we have a chance to define what that tag should be—so let's carefully pick a single one, which will hopefully influence what other software does.

Reply:
Went with the form of `ARTISTS_CREDIT`, and `ArtistsCredit` for ASFStorageStyle. I don't have a strong opinion here so happyp to change it